.expeditor/publish-release-notes.sh

```diff
@@ -30,6 +30,12 @@ pushd ./automate.wiki
 ## Backward Incompatibilities
 -
 
+## Chef Product Versions
+
+This release depends on:
```

Comment:
is uses a better word here? im not sure
do we want the bldr version too?

Reply:
oh theres also chef infra server right?

Reply:
@susanev I'm happy to add those two. Anything else before I add another commit?

Reply:
well hold on, i kinda want @vjeffrey or someone else to confirm for me @ryancragun thoughts?